engines/terraform/plugins/gcpiamserviceaccount/module/variables.tf

```diff
@@ -0,0 +1,18 @@
+variable "nitric" {
+  type = object({
+    name = string
+    stack_id = string
```

Comment:
if not already included we'll need to add this to the `terraform.go` which we can do in a separate PR, but this is a good catch